src/components/counter.component.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+
+class Counter extends Component {
+  state = { count: 0 }
+
+  handleIncrease = () => {
+    this.setState({ count: this.state.count + 1 })
```

Comment:
if using the previous state to set the next state, it's better to use the state callback in `setState`:

```javascript
this.setState(state => ({ count: state.count + 1 }))
```